crates/block-producer/Cargo.toml

```diff
@@ -11,19 +11,12 @@ readme.workspace = true
 homepage.workspace = true
 repository.workspace = true
 
-[[bin]]
-name = "miden-node-block-producer"
-path = "src/main.rs"
-bench = false
-doctest = false
-
 [features]
 tracing-forest = ["miden-node-utils/tracing-forest"]
 
 [dependencies]
 anyhow = { version = "1.0" }
```

Comment:
Using `anyhow` or `eyre` in libraries is considered as a bad practice, because it's hard to handle different errors by library users. But binary is a different case: we usually just print error message from the binary to the user and don't need to handle it, so `anyhow` is suitable there.